GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -1280,6 +1310,12 @@ void DisposeRevisionReader()
             }
         }
 
+        /// <summary>
+        /// The parents for commits are replaced with the parent in the graph (as all commits may not be included)
+        /// See https://git-scm.com/docs/git-log#Documentation/git-log.txt---parents
+        /// </summary>
+        public bool ParentsAreRewritten { get; private set; } = false;
```

Comment:
Why is this public?

Reply:
After the latest changes, it can be private, the logic is now in GetActualRevision()
(I realized this after pushing the latest update, put a note about it.